packages/components/src/horizontal-bar-list/stats-hero-card.tsx

```diff
@@ -71,7 +72,10 @@ const StatsHeroCard = ( {
 				className={ `${ BASE_CLASS_NAME }-header ${ headerClassName } ${ BASE_CLASS_NAME }-header--split` }
 			>
 				<div className={ `${ BASE_CLASS_NAME }-header--main` }>
-					{ titleNode }
+					<div className={ `${ BASE_CLASS_NAME }-header--main__left` }>
```

Comment:
Just a note that this stats-hero-card component is only being used by the Stats Locations module, since it is the only module. using splitHeader + heroElement:

https://github.com/Automattic/wp-calypso/blob/989e305e1cebaaab5850bca6acf2a4c9842e0cea/packages/components/src/horizontal-bar-list/stats-card.tsx#L14-L17